apps/dashboard/app/models/jobs/project.rb

```diff
@@ -0,0 +1,19 @@
+class Jobs::Project
+  def ls(dir)
+    Pathname.new(dir).each_child.map do |path|
+      Jobs::Project.stat(path)
+    end.sort_by { |p| p[:directory] ? 0 : 1 }
+  end
+
+  def self.stat(dir)
+    path = Pathname.new(dir)
+      {
+        name: path.basename,
+        directory: path.stat.directory?
+      }
+  end
+
+  def pwd
+    Dir.pwd
+  end
+end
```

Comment:
Note quite sure what's up with this model, but it mostly being static isn't likely what we want. 

Seems like it should be something like this.  Where once you can a project, it encapsulates all the information about the project like files and so on.
```ruby
class Jobs::Project
  self << class
    def all
      # return all 'projects' in ~/ondemand/data/jobs
    end
  end

  attr_reader  :dir

  def initialize(dir)
    @dir = dir.to_s
  end

  def files
     # return all the files in dir
  end
end
```

Reply:
I was just trying to get some files mocked up in the view quickly was all. The actual model is something we had said would be deferred until after the views.